javascript/javascript1/week4/java-week-4.js

```diff
@@ -0,0 +1,86 @@
+const list = [];
+let userName = "";
```

Comment:
Hej Lokesh . good work.:)

Reply:
Thank you for your informative and constructive feedback. I will make required changes and update homework 🙂